activesupport/lib/active_support/message_pack.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+begin
+  require "msgpack"
```

Comment:
Noted!  Also, if we're going to use a `Pool` (as per https://github.com/rails/rails/pull/47770#discussion_r1148442212) and have a leading signature (as per https://github.com/rails/rails/pull/47770#discussion_r1148440069), I think we'll need whichever next version of `msgpack` includes https://github.com/msgpack/msgpack-ruby/commit/a41195c17758ec967c98a35a03824229e39161ca.

Reply:
Yeah, we've tracking down a weird lost pointer issue since about two weeks that can cause segfaults, as soon as we have it figured it out I'll release 1.7.

Reply:
I released `1.7.0` last week.